### DISCUSSION TITLE
DAI 2024-2025 - Practical work 2 - Wulliamoz Nathan, Kocher Benjamin

### DISCUSSION THREAD

**Discussion by BKo1706** [heig-vd-dai-course/heig-vd-dai-course#481] [Show and tell]
Notre idée est de faire une messagerie, avec la possibilité de crée des rooms sécurisées par un mot de passe. 
A l'intérieur de ces rooms, tous les messages sont cryptés à l'aide du mot de passe de la room afin d'empêcher l'interception de messages en clair.

En plus de cela et si nous avons le temps, nous aimerions stocker sous formes de fichiers externes les différentes conversation dans les rooms.

Lien du repo git : https://github.com/nathan-heig/DAI-16-practical-work-2.git
Hash du dernier commit : 468d5078adad4abe09a048d38039dd508651c428

**Comment by ludelafo** [comment]
Tout bon pour nous !
